bigcache.go

```diff
@@ -83,6 +83,13 @@ func (c *BigCache) Set(key string, entry []byte) error {
 	return shard.set(key, hashedKey, entry)
 }
 
+// Del removes the key
+func (c *BigCache) Del(key string) error {
```

Comment:
Can we change "Del" to "Delete"? Shortcut can be the source of misunderstanding or doubts about what function does when we see only the name. Therefore I prefer full name over shortcut, especially when the function is exposed in the API. Beside that I don't have any suggestions, the code looks great, thanks for this PR.